content/blog/2022-06-14-DVC-VS-Code-extension.md

```diff
@@ -0,0 +1,169 @@
+  CLI
+
+As data scientists, DVC is our toolbox. This extension turns VS Code into our
+workshop.
+
+# Features
+
+Here are the main features that are available when you install the extension
+right now:
+
+## DVC View: experiments, parameters, metrics, and plots
+
+The main feature of our extension is the DVC view, which provides a one-stop
+shop for everything related to your machine learning experiments. You can run
+new experiments from here, manage parameters, and compare both metrics and plots
+for different models. Never has it been easier to analyze the results of your
+experiments and find the best model.
+
+<!--
+![Experiment tables to compare different
+experiments](/uploads/images/2022-06-14/dvc-view-experiments-table.png)
+
+![Plots to visualize your
+experiments](/uploads/images/2022-06-14/dvc-view-plots.png) -->
+
+<video controlslist="nodownload" preload="metadata" autoplay muted loop
+style="width:100%;">
+
+  <source src="/uploads/images/2022-06-14/run-experiment.mp4" type="video/mp4">
+Your browser does not support the video tag.
+</video>
+
+_Quickly run new ML experiments and view them in the experiments table._
+
+<video controlslist="nodownload" preload="metadata" autoplay muted loop
+style="width:100%;">
+
+  <source src="/uploads/images/2022-06-14/compare-experiments.mp4" type="video/mp4">
+Your browser does not support the video tag.
+</video>
+
+_Compare experiments with each other._
+
+<video controlslist="nodownload" preload="metadata" autoplay muted loop
+style="width:100%;">
+
+  <source src="/uploads/images/2022-06-14/live-metrics.mp4" type="video/mp4">
+Your browser does not support the video tag.
+</video>
+
+_Live updates to plots when using DVCLive._
+
+## Source control management
+
+On top of Git version control for your code, the source control view now lets
+you manage datasets and models tracked by DVC without using the terminal. The
+DVC panel shows you the state of the workspace, similarly to the familiar Git
+panel. From here you can track artifacts and synchronize versions with your
+remote repository.
+
+![Get a quick overview of changes to your data in the source control
+panel](/uploads/images/2022-06-14/source-control.png)
+
+## DVC tracked explorer
+
+Resources tracked by DVC can be viewed and managed through the tracked explorer
+view. See at a glance which datasets and models have been changed, and navigate
+through all files contained in the DVC project(s) of your workspace.
+
+## Command palette
+
+The DVC extension makes extensive use of the Command Palette, reducing the need
+to learn syntax by heart. Want to run a new experiment or pull data from your
+remote repository? Give the word in the Command Palette and the DVC extension
+will guide you along all required details.
+
+---
+
+https://media.giphy.com/media/cEYFeE4wJ6jdDVBiiIM/giphy.gif
+
+# What's next?
+
+We're excited that our VS Code extension is live and available to the public!
```

Comment:
This sentence is unnecessary.  I think it better to remove and start this section with the next sentence.